### ISSUE TITLE
Conftest using specific .rego file is not working properly

### ISSUE THREAD

**Issue by rawc0der** [open-policy-agent/conftest#383]
When using specific `.rego` file passed through command line arguments, the test always passes, disregarding any verification rules. 

> (from the manpage)
> -p, --policy string   path to the Rego policy files directory. For the test command, specifying a specific .rego file is allowed. (default "policy")

Steps to reproduce:
```
git clone https://github.com/open-policy-agent/conftest/
cd examples/kubernetes
```
I am using latest (as of this writing) **conftest Version: 0.20.0**. In the default examples folder from this repository, the `policy/` folder contains multiple `.rego` files.  Using any of these policies independently will result in the test always passing.

FAULTY Examples:
```
$ conftest test deployment.yaml -p policy/labels.rego                                                                                                                                             
1 test, 1 passed, 0 warnings, 0 failures, 0 exceptions
```
```
$ conftest test deployment.yaml -p policy/deny.rego                                                                                                                                             
2 tests, 2 passed, 0 warnings, 0 failures, 0 exceptions
```

WORKING Example:
```
$ conftest test deployment.yaml -p policy/                                                                                                                                                        
FAIL - deployment.yaml - Containers must not run as root in Deployment hello-kubernetes
FAIL - deployment.yaml - Deployment hello-kubernetes must provide app/release labels for pod selectors
FAIL - deployment.yaml - hello-kubernetes must include Kubernetes recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
FAIL - deployment.yaml - Found deployment hello-kubernetes but deployments are not allowed

5 tests, 1 passed, 0 warnings, 4 failures, 0 exceptions
```
It is obvious from the WORKING example that the tests should be failing.
Validating against specific `.rego` policies is not working properly, please check!

**Comment by Blokje5** [comment]
Thanks for reporting the issue @rawc0der! Could you share the policies you used? I'll see if I can reproduce the issue.

**Comment by jpreese** [comment]
This looks to be working as intended. In the given example, both the `deny.rego` and `labels.rego` policies depend on a result from `kubernetes.rego`. Since you're not including `kubernetes.rego` in your `-p` argument, those rules are returning undefined, which will ultimately return as successful.

In conftest `0.21.0` you can specify multiple policy files so you could do:

`conftest test deployment.yaml -p policy/deny.rego -p policy/kubernetes.rego`

Which will return two failures.

**Comment by rawc0der** [comment]
Thanks for the reply @jpreese and @Blokje5! Think the problem was that in version `0.20.0` the policy files are not merged correctly when adding multiple sources with `-p` argument.

Closing this issue, as I see it is resolved in version `0.21.0` and merged in #344 .
